pkgs/applications/office/jabref/default.nix

```diff
@@ -131,19 +72,13 @@ stdenv.mkDerivation rec {
     python3
   ];
 
-  buildPhase = ''
-    runHook preBuild
-
-    export GRADLE_USER_HOME=$(mktemp -d)
-    gradle \
-      --offline \
-      --no-daemon \
-      -PprojVersion="${version}" \
-      -PprojVersionInfo="${version} NixOS" \
```

Comment:
Why do you remove this line?

Reply:
look at `gradleFlags` and `preConfigure`